src/main/java/step2/model/WinningLotto.java

```diff
@@ -1,23 +1,39 @@
 package step2.model;
 
 import java.util.List;
-import java.util.function.Predicate;
-import java.util.stream.Collectors;
 
 public class WinningLotto {
 
-	private List<Integer> winNumList;
+	private static final int SECOND_MATCH_NUM = 5;
+	private List<Integer> winLotto;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 좋을 것 같아요.
```suggestion
	private static final int SECOND_MATCH_NUM = 5;
	
	private List<Integer> winLotto;
```